coding-concepts.js

```diff
@@ -7,48 +7,67 @@
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
 const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+//console.log(colors.push("indigo"))
 
 // a) Your answer:
+//the console will log the the amount of indices
+
 // b) Verify and explain:
+//the console loggged 5 because <.push()> returns a number
+//which would be the total number of colors in the array
 
 // --------------------1) What will this log?
 
 const cohort = "LEARN 2023"
-// console.log(cohort.length)
+//console.log(cohort.length)
 
 // a) Your answer:
+//This will log 10 
+
 // b) Verify and explain:
+//The console logged 10 because there are 10 characters
 
 // --------------------2) What will this log?
 
 const greeting = "Hello World!"
-// console.log(greeting[4])
+//console.log(greeting[4])
 
 // a) Your answer:
+//The console will log 'o'
+
 // b) Verify and explain:
+//The console logged 'o' because 'o' is the 4th index of the variable <greeting>
 
 // --------------------3) What will this log?
 
 const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
-// console.log(languages[index])
+//console.log(languages[index])
 
 // a) Your answer:
+//The console will log "Ruby"
+
 // b) Verify and explain:
+//The console logged "Ruby" because it is the 1st index of the array <languages>
```

Comment:
I think I know what you meant here, just keep in mind the 1st index would be index 0.